annotations/annotations_api.go

```diff
@@ -36,7 +36,12 @@ const (
 	nextVideoAnnotationLifecycle = "next-video"
 )
 
-// UPPError encapsulate error information for errors originating from calls to UPP annotations endpoint.
+type UPP1Error struct {
+	msg     string
```

Comment:
**[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
`msg` is unused (structcheck)